### PR TITLE
Fix hang in ipc.exe with invalid arguments

### PR DESCRIPTION
**Pull request by andygraham**
Also on successful compilation  "Saved to .." now shows correct file type

### CODE REVIEW

File: Languages/IronPython/IronPythonCompiler/Program.cs [hunk 1]
```diff
@@ -272,17 +272,27 @@ class Program {
...
+                    Path.ChangeExtension(config.Output, ".dll"), 
+                    compileOptions, 
+                    config.Files.ToArray());
+                
+                var outputfilename = Path.ChangeExtension(config.Output, ".dll");
+                if (config.Target != PEFileKinds.Dll) {
+                    outputfilename = Path.ChangeExtension(config.Output, ".exe");                    
+                    GenerateExe(config);
+                }
+                ConsoleOps.Info("Saved to {0}", config.Output);
```
Comment (slide) [inline Languages/IronPython/IronPythonCompiler/Program.cs]: Doesn't this need to use outputfilename instead of config.Output?

Comment (andygraham) [reply Languages/IronPython/IronPythonCompiler/Program.cs]: Yes, sorry, and I thought I carefully checked it line by line after I tested it :( which I had to do separately as the 2.7.7 source code doesn't compile under VS2013 which is what I use - though 2.7.5 did as I recall.

Comment (andygraham) [reply Languages/IronPython/IronPythonCompiler/Program.cs]: Yes, sorry, and I thought I carefully checked it line by line after I tested it :( which I had to do separately as the 2.7.7 source code doesn't compile under VS2013 which is what I use - though 2.7.5 did as I recall.

Comment (andygraham) [reply Languages/IronPython/IronPythonCompiler/Program.cs]: Sorry for the multiple comments. The page seemed to freeze after the first try so I pressed Enter again and it seemed to freeze again but both on refreshing the page both entries had made it

**Pull request merged**